dl/src/main/scala/com/intel/analytics/bigdl/optim/DistriOptimizer.scala

```diff
@@ -514,7 +514,7 @@ class DistriOptimizer[T: ClassTag] private[optim](
     optimMethod.clearHistory(state)
     state("dropPercentage") = dropPercentage
     state("warmupIterationNum") = warmupIterationNum
-    state("comupteThresholdbatchSize") = comupteThresholdbatchSize
+    state("comupteThresholdbatchSize") = computeThresholdbatchSize
```

Comment:
state("comupteThresholdbatchSize") -> state("computeThresholdbatchSize")? Or some unit tests will be broken